src/Driver/Driver.php

```diff
@@ -693,4 +691,24 @@ protected function getDSN(): string
     {
         return $this->options['connection'] ?? $this->options['dsn'] ?? $this->options['addr'];
     }
+
+    /**
+     * Creating a context for logging
+     *
+     * @param float             $queryStart Query start time
+     * @param PDOStatement|null $statement  Statement
+     *
+     * @return array
+     */
+    protected function createLoggerContext(float $queryStart, ?PDOStatement $statement): array
```

Comment:
maybe `getLoggerContext`? or `defineLoggerContext`? I think this name is fine, but `create` confuses a bit

Reply:
@wolfy-j Thanks for paying attention, corrected to defineLoggerContext